frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -50,17 +56,70 @@ const fieldsFor = (providedAPI: CRDDescription) => _.get(providedAPI, 'specDescr
 })) as OperandField[];
 
 export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
-  type FormValues = {[path: string]: any};
-  const [formValues, setFormValues] = React.useState<FormValues>({'metadata.name': 'example', 'metadata.labels': []});
-  const [error, setError] = React.useState();
-
   const fields: OperandField[] = !_.isEmpty(props.clusterServiceVersion)
     ? fieldsFor(providedAPIsFor(props.clusterServiceVersion).find(desc => referenceForProvidedAPI(desc) === referenceForProvidedAPI(props.providedAPI)))
     : [];
 
+  const defaultValueFor = (field: OperandField) => {
+    if (field.capabilities.includes(SpecCapability.podCount)) {
+      return 1;
+    }
+    if (field.capabilities.includes(SpecCapability.resourceRequirements)) {
+      return {limits: {cpu: '', memory: ''}, requests: {cpu: '', memory: ''}};
+    }
+    if (field.capabilities.includes(SpecCapability.password)) {
+      return '';
+    }
+    if (field.capabilities.some(c => c.startsWith(SpecCapability.k8sResourcePrefix))) {
+      return null;
+    }
+    if (field.capabilities.includes(SpecCapability.checkbox)) {
+      return false;
+    }
+    if (field.capabilities.includes(SpecCapability.booleanSwitch)) {
+      return false;
+    }
+    if (field.capabilities.includes(SpecCapability.imagePullPolicy)) {
+      return ImagePullPolicy.IfNotPresent;
+    }
+    if (field.capabilities.includes(SpecCapability.updateStrategy)) {
+      return 'Rolling';
+    }
+    if (field.capabilities.includes(SpecCapability.text)) {
+      return '';
+    }
+    if (field.capabilities.includes(SpecCapability.number)) {
+      return '';
```

Comment:
Is it expected that `SpecCapability.number` returns a string?

Reply:
No, but we can't pass `null` to the `input` as a default value or it complains. We use `_.toInteger()` and remove nil values on create, so we should be fine.